src/data/agile-quiz.ts

```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
       "Story points are used to estimate the amount of effort required to deliver a user story for a project.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/#story-points-and-complexity-points",
   },
+  {
+    Questions:
+      "What is an iterative development methodology, where both development and testing activities are concurrent.",
```

Comment:
This wording is a little to close to here
https://sceweb.sce.uhcl.edu/helm/WEBPAGE-AgileTesting


```suggestion
      "What is the name for the process that focuses on continuously building, testing, and gathering feedback for software products?",
```